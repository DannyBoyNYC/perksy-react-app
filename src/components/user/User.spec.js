const React = require('react');

const User = require('./index').default;

describe('<User.* />', () => {
  let component;
  const requiredProps = {
    brandName: 'Perksy',
    firstName: 'Jane',
    lastName: 'Doe'
  };

  describe('<User.Avatar />', () => {
    beforeEach(() => {
      component = shallowUntilTarget(
        <User.Avatar {...requiredProps} />,
        'User'
      );
    });

    it('should render correctly with all required props', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
