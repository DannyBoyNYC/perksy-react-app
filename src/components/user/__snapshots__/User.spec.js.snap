// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<User.* /> <User.Avatar /> should render correctly with all required props 1`] = `
<div
  className="User-root-0-1-1"
>
  <img
    alt=""
    className="User-image-0-1-2"
  />
  <div
    className="User-name-0-1-3"
  >
    Jane Doe
  </div>
  <div
    className="User-brandName-0-1-4"
  >
    Perksy
  </div>
</div>
`;
